arangod/GeneralServer/AcceptorTcp.cpp

```diff
@@ -114,13 +114,13 @@ void AcceptorTcp<T>::close() {
     _open = false;  // make sure the _open flag is `false` before we
                     // cancel/close the acceptor, since otherwise the
                     // handleError method will restart async_accept.
-    _acceptor.close();
+    _ctx.io_context.wrap([this]() { _acceptor.close(); });
```

Comment:
Not saying anything against this PR, but should we also do the same changes in `AcceptorUnixDomain` in this PR or a separate one?
It looks like it could also be affected by the same issues.